mission-control-client/src/components/dashboard/projects/Project.js

```diff
@@ -5,16 +5,16 @@ const Project = props => {
 
   return (
     <div
-      onClick={() => props.setActiveProduct(props.el)}
+      onClick={() => props.setActiveProduct(props.el.project)}
       id={
```

Comment:
@agordon1123 should this be props.setActiveProject?

Reply:
I am confused as well with that

Reply:
Correction made. Thank you!